scipy/optimize/_linprog_util.py

```diff
@@ -1372,18 +1319,18 @@ def _postsolve(x, postsolve_args, complete=False, tol=1e-8, copy=False):
     # if "complete", problem was solved in presolve; don't do anything here
     if not complete and bounds is not None:  # bounds are never none, probably
         n_unbounded = 0
-        for i, b in enumerate(bounds):
+        for i in range(bounds.shape[0]):
```

Comment:
Is this necessary or preferred style?  It is easier to review if you start by changing only the essentials, then we tackle optional thigs if desired.

Reply:
You are right, the changes here are not necessary.
I used `range()` to explicitly show we iterate over the rows of the array, which is not obvious when using `enumerate()`. I changed the indentation of the if-statement also for readability only.